environments/prod/prod.tfvars

```diff
@@ -449,6 +449,13 @@ frontends = [
         "942400",
       ]
     }
+    global_exclusions = [
```

Comment:
needs to be done in the other 3 places too

Reply:
Yes - done now